src/frontend/Components/HomePage/HomePage.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import london from './londonskyline.jpg';
+import './HomePage.css';
+
+class HomePage extends React.Component {
+  render() {
+    return (
+      <div className='header'>
```

Comment:
You don't need the div with a header class here, its just an extra div thats not doing anything at the moment